Source/Core/Core/PowerPC/Interpreter/Interpreter_Tables.cpp

```diff
@@ -147,14 +147,14 @@ static GekkoOPTemplate table19[] =
 {
 	{528, Interpreter::bcctrx,      {"bcctrx", OPTYPE_BRANCH, FL_ENDBLOCK, 1, 0, 0, 0}},
 	{16,  Interpreter::bclrx,       {"bclrx",  OPTYPE_BRANCH, FL_ENDBLOCK, 1, 0, 0, 0}},
-	{257, Interpreter::crand,       {"crand",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{129, Interpreter::crandc,      {"crandc", OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{289, Interpreter::creqv,       {"creqv",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{225, Interpreter::crnand,      {"crnand", OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{33,  Interpreter::crnor,       {"crnor",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{449, Interpreter::cror,        {"cror",   OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{417, Interpreter::crorc,       {"crorc",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{193, Interpreter::crxor,       {"crxor",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
+	{257, Interpreter::crand,       {"crand",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{129, Interpreter::crandc,      {"crandc", OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{289, Interpreter::creqv,       {"creqv",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{225, Interpreter::crnand,      {"crnand", OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{33,  Interpreter::crnor,       {"crnor",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{449, Interpreter::cror,        {"cror",   OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{417, Interpreter::crorc,       {"crorc",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{193, Interpreter::crxor,       {"crxor",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
```

Comment:
Do these instructions still need FL_EVIL given that we have FL_SET_CRn?  Or do we also need FL_IN_CR or something like that?  (It seems like a bad idea to have our generic "don't optimize this" flag attached to a commonly used instruction.)